app/index.jade

```diff
@@ -176,9 +179,27 @@ head
   
   <!-- @if PRODUCTION !>
   link(rel='stylesheet', href='css/application.css')
-  script(src='js/application.min.js')  
   <!-- @endif -->
+
+body(ng-controller="AppCtrl", ui-view="body")
   
+  //- This does not depend on Angular and will show as soon as the images are done loading.
+  <!-- @if PRODUCTION !>
+  .blocket-loading
+    div
+      img.full(src="/img/blocket-full-4847e61888d2d3c9f33f14840f92783d1c91dea2.png")
+      img(src="/img/blocket-no-boosters-c3f2a027ddfdbbce8eba8fc5da84c65d89d14fbc.png")
```

Comment:
No need to disable this in non-production. Also no need to put the sha hash in the image url's. This is handled by `grunt dist_unsafe` / `grunt release`.